app/backend/src/couchers/servicers/communities.py

```diff
@@ -244,26 +244,39 @@ def RemoveAdmin(self, request, context, session):
 
     def ListMembers(self, request, context, session):
         page_size = min(MAX_PAGINATION_LENGTH, request.page_size or MAX_PAGINATION_LENGTH)
-        next_member_id = int(request.page_token) if request.page_token else 0
+        page_number = request.page_number or 1
+        offset = (page_number - 1) * page_size
+
         node = session.execute(select(Node).where(Node.id == request.community_id)).scalar_one_or_none()
+
         if not node:
             context.abort(grpc.StatusCode.NOT_FOUND, errors.COMMUNITY_NOT_FOUND)
+
+        total_items = session.execute(
+            select(func.count(User.id))
+            .select_from(User)
+            .join(ClusterSubscription, ClusterSubscription.user_id == User.id)
+            .where_users_visible(context)
+            .where(ClusterSubscription.cluster_id == node.official_cluster.id)
+        ).scalar_one()
+
         members = (
             session.execute(
                 select(User)
```

Comment:
Can we use `LiteUser` on the backend here?

Reply:
Yes